project-base/storefront/components/Pages/Wishlist/Wishlist.tsx

```diff
@@ -37,59 +37,57 @@ export const Wishlist: FC = () => {
     }
 
     return (
```

Comment:
Isn't the wishlist wrapped in two Weblines now?

Reply:
Ah, yup. I've removed Webline only from SharedWishlist 🙈 Removed from here. https://github.com/shopsys/shopsys/pull/2836/files#diff-afa93232f78935a789a76cf48399470385db0cb416922ab619f92c820cf72070